### PR TITLE
Update move examples

### PR DESCRIPTION
**Pull request by wswebcreation**
I've been struggling the last days with the `move` command because the `moveTo` implies an absolute position, but it's just a relative offset, see also https://github.com/appium/appium/issues/6257#issuecomment-196949576

This PR hopefully makes it more clear

### CODE REVIEW

File: docs/en/commands/interactions/touch/move.md [hunk 1]
```diff
@@ -24,9 +24,17 @@ actions.perform()
...
+// Move to 200,200 (as in, an additional 100 in each direction)
 driver.touchMultiPerform([
-  { action: 'press', options: { x: 100, y: 250 }},
-  { action: 'moveTo', options: { x: 300, y: 100 }},
+  { action: 'press', options: { x: 100, y: 100 }},
+  { action: 'moveTo', options: { x: 100, y: 100 }},
+  { action: 'release' }
+]);
+
+// Move to // Move to 0,0 (as in, an additional -100 in each direction)
```
Comment (jeaye) [inline docs/en/commands/interactions/touch/move.md]: We've a double comment here.

**Comment by jeaye** [comment]
This should save people a lot of time. Thank you, @wswebcreation!

**Review by jeaye — CHANGES_REQUESTED** [review]

File: docs/en/commands/interactions/touch/move.md (hunk 1 above)
Comment (wswebcreation) [reply docs/en/commands/interactions/touch/move.md]: Fixed it

**Comment by tfcardoso** [comment]
@wswebcreation you sure? I was under the impression that behavior had been changed by @mykola-mokhnach in https://github.com/appium/appium-android-driver/pull/292

**Comment by mykola-mokhnach** [comment]
Yes the moveTo coordinates should be absolute according to the spec. However, previous appium version had a bug, where these coordinates were considered as relative.

**Comment by wswebcreation** [comment]
Ok, so if I understand correctly this will be fixed in 1.8.0?

Meaning that this PR is only for the versions < 1.8.0

Maybe we should still tell something about it in the docs, I've seen a lot of questions in the webdriver.io channel concerning this part

**Review by jeaye — APPROVED** [review]

**Comment by dpgraham** [comment]
Sorry for the late review on this one.

This .md file is actually auto-generated and the actual file that needs to be edited is: https://github.com/appium/appium/blob/master/commands-yml/commands/interactions/touch/move.yml

@wswebcreation Do you want to transfer your changes over to the .yml file or do you want me to do that?

**Comment by wswebcreation** [comment]
Hi @dpgraham

I’d like to but is this still valid. Because when I look at the remark of @mykola-mokhnach I do wonder.

**Comment by jlipps** [comment]
@dpgraham we should put a big invisible comment at the top of these files that they are auto-generated and shouldn't be edited by hand.

**Comment by dpgraham** [comment]
@jlipps Yeah, that's a good idea

**Pull request closed without merging**